CHANGELOG.rst

```diff
@@ -29,6 +29,11 @@ Added
 
 Changed
 -------
+- Do not retrain the entire Core model if only the ``templates`` section of the domain is changed.
```

Comment:
@btotharye Please check these diffs. They shouldn't be there.

Reply:
sorry didn't notice this fixing it now